.github/workflows/release-brew.yaml

```diff
@@ -3,21 +3,118 @@ on:
   release:
```

Comment:
Could we add a comment (at the top of this file, not here) to guide people who understand GitHub workflows but not Homebrew?  Taps are here:

```
# See documentation for Homebrew taps (third-party repositories): https://docs.brew.sh/Taps
```

Is there other documentation we could point to that explains the process of creating bottles, etc?  Also, could you reproduce your nice explanation of what this workflow does that you included in the PR comment?  In particular, that it checks syntax and tests both the package and the bottle.  Maybe point to the Homebrew standards that you are following and say "We follow the Homebrew best practices at XXXX".